install_config/persistent_storage/persistent_storage_nfs.adoc

```diff
@@ -11,67 +11,64 @@
 toc::[]
 
 == Overview
-You can provision your OpenShift cluster with
-link:../../architecture/additional_concepts/storage.html[persistent storage] using
-https://access.redhat.com/documentation/en-US/Red_Hat_Enterprise_Linux/7/html/Storage_Administration_Guide/ch-nfs.html[NFS].
-Some familiarity with Kubernetes and NFS is assumed.
 
-The Kubernetes
-link:../../architecture/additional_concepts/storage.html[persistent volume]
-framework allows administrators to provision a cluster with persistent storage
-and gives users a way to request those resources without having any knowledge of
-the underlying infrastructure. Persistent volumes are not bound to a single
-project or namespace; they can be shared across the OpenShift cluster.
-link:../../architecture/additional_concepts/storage.html#persistent-volume-claims[Persistent
-volume claims], however, are specific to a project or namespace and can be
-requested by users.
-
-For a detailed example, see the guide for
-https://github.com/openshift/origin/tree/master/examples/wordpress[WordPress and
-MySQL using NFS].
-
-[IMPORTANT]
+OpenShift clusters can be provisioned with
+link:../../architecture/additional_concepts/storage.html[persistent storage]
+using NFS. Persistent volumes (PVs) and persistent volume claims (PVCs) provide
+a convenient method for sharing a volume across a project. While the
+NFS-specific information contained in a PV definition could also be defined
+directly in a pod definition, doing so does not create the volume as a distinct
+cluster resource, making the volume more susceptible to conflicts.
+
+This topic covers the specifics of using the NFS persistent storage type. Some
+familiarity with OpenShift and
+https://access.redhat.com/documentation/en-US/Red_Hat_Enterprise_Linux/7/html/Storage_Administration_Guide/ch-nfs.html[NFS]
+is beneficial. See the
+link:../../architecture/additional_concepts/storage.html[Persistent Storage]
+architecture topic for details on the OpenShift persistent volume (PV) framework
+in general.
+
+[NOTE]
 ====
-High-availability of storage in the infrastructure is left to the underlying
-storage provider.
+_Project_ and _namespace_ are used interchangeably throughout this document. See
+link:../../architecture/core_concepts/projects_and_users.html#namespaces[Projects
+and Users] for details on the relationship.
```

Comment:
@jeffvance Thanks. Yeah, I get the reasoning, and we don't hide "namespace" completely (https://docs.openshift.com/enterprise/latest/architecture/core_concepts/projects_and_users.html#namespaces).

I have just (in general, not just in this PR) wondered if it's worth liberally mixing the terms versus sticking with the OpenShift parlance (project, in this case) where we're able, to keep things simpler. We want readers generally to think in terms of projects, overall. I only saw one spot in these topics where "namespace" needed specifically to be used (a reference to the "namespace:" field, which could be explained in its callout description). Everywhere else, "project" looked like it would work / make sense just as well. Even the `oc export ns` looks like it could safely be `oc export project` (shows the same bits of info we're interested in here, AFAICT).

I think they're uncommon enough that for the spots where "namespace"-specific naming would need to be shown or used (e.g., "use --all-namespace to affect all projects"), we could just have an explanation or "Note" there as needed.